suites/message/paych.go

```diff
@@ -139,14 +139,17 @@ func MessageTest_Paych(t *testing.T, factory state.Factories) {
 		settleResult := td.ApplyOk(
 			td.MessageProducer.PaychSettle(receiver, paychAddr, nil, chain.Value(big_spec.Zero()), chain.Nonce(0)))
 
+		td.AssertActor(receiver, initialBal, settleResult.Receipt, 1)
+
 		// advance the epoch so the funds may be redeemed.
 		td.ExeCtx.Epoch += paych_spec.SettleDelay
 
+		prevBal := td.GetBalance(receiver)
 		collectResult := td.ApplyOk(
 			td.MessageProducer.PaychCollect(receiver, paychAddr, nil, chain.Nonce(1), chain.Value(big_spec.Zero())))
 
 		// receiver_balance = initial_balance + paych_send - settle_paych_msg_gas - collect_paych_msg_gas
-		td.AssertBalance(receiver, big_spec.Sub(big_spec.Sub(big_spec.Add(toSend, initialBal), settleResult.Receipt.GasUsed.Big()), collectResult.Receipt.GasUsed.Big()))
+		td.AssertActor(receiver, big_spec.Add(prevBal, toSend), collectResult.Receipt, 2)
```

Comment:
To be clear, I'm expecting/requesting this to be `td.AssertActor(receiver, prevBal, toSend, collectResult.Receipt, 2)`, pushing addition down, so that is not passing `prev + sent` as something called `prevBalance`.